### PR TITLE
[FIX] l10n_ar_tax: impuesto P. IIBB CABA 3.0% sin jurisdicción y error de runbot

### PR DESCRIPTION
**Pull request by pablohmontenegro**
Ticket: 87219

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/1038).

File: l10n_ar_tax/models/account_fiscal_position_l10n_ar_tax.py [hunk 1]
```diff
@@ -90,6 +90,7 @@ def _ensure_tax(self, rate):
                 'amount': rate,
                 'active': True,
                 'name': name,
+                'l10n_ar_state_id': self.default_tax_id.l10n_ar_state_id,
```
Comment (zaoral) [inline l10n_ar_tax/models/account_fiscal_position_l10n_ar_tax.py:93]: not sure why this is needed, because the l10n_ar_state_id will be automatically copy when duplicating a tax.

File: l10n_ar_tax/models/account_chart_template.py [hunk 2]
```diff
@@ -95,7 +95,7 @@ def _add_wh_taxes(self, company):
 
         profits_domain = withholdings_domain + [('l10n_ar_tax_type', 'in', ['earnings', 'earnings_scale'])]
         sequence = self.env['ir.sequence'].create({
-                'name': tax.invoice_label or 'Retención de Ganancias',
+                'name': 'Retención de Ganancias',
```
Comment (zaoral) [inline l10n_ar_tax/models/account_chart_template.py:98]: invoice label is not set on any taxes.
Is not used on any of the data/template tax csv files
maybe the problem is that we need to complete that info there?

File: l10n_ar_tax/__init__.py [hunk 3]
```diff
@@ -23,7 +23,7 @@ def _l10n_ar_update_taxes(env):
     """
 
     # si tiene instalado chart ri o exento le actualizamos impuestos
-    companies = env['res.company'].search([('chart_template', 'in', ('ar_ri', 'ar_ex'))])
+    companies = env['res.company'].search([('chart_template', 'in', ('ar_base', 'ar_ri', 'ar_ex'))])
```
Comment (zaoral) [inline l10n_ar_tax/__init__.py:26]: monotribuista copany (coa ar_base) is archived, maybe this is the reason why is not applying the jurisdiction?

File: l10n_ar_tax/models/account_fiscal_position_l10n_ar_tax.py [hunk 4]
```diff
@@ -105,6 +106,16 @@ def _get_tax_from_ws(self, partner, date):
             tax = self._ensure_tax(aliquot)
         # por mas que sea no inscripto creamos partner aliquot porque si no en cada
         # nueva linea o cambio se conecta a ws
+        if self.env.ref('base.user_demo', raise_if_not_found=False):
```
Comment (zaoral) [inline l10n_ar_tax/models/account_fiscal_position_l10n_ar_tax.py:109]: I think that the problem could be here
https://github.com/ingadhoc/odoo-argentina/blob/18.0/l10n_ar_tax/models/account_chart_template.py#L142-L143 The last time we had an error related to duplicated taxes was because there were trying load() the chart of account

Also, testing PR's runbot found that the duplicates persist with your changes, leave an image bellow 
image

I found these commits that maybe could be related to the tax duplication, please check them
* https://github.com/odoo/odoo/commit/a2f503048d54cfca15d0d029909160bfa0e31afc
* https://github.com/odoo/odoo/commit/b7d27ea5c89da0156e5963537e058d7d0cd998c0#diff-af1d0564587496a753998da304bbd397352d0ca5811f273a4bf0ad0113d2ab55

**Review by zaoral — COMMENTED** [review]
Hi @pablohmontenegro 

Thank you for the PR.
I see that the changes resolve runbot errors, and partially do not generate changes more than in demo data.

I will merge this PR because we need it and also the team, but I am not completely sure about it, I left you some comments so you can check it on Monday and maybe we can do a new PR with the fixes.

Best Regards!

**Comment by zaoral** [comment]
@roboadhoc r+

File: l10n_ar_tax/models/account_chart_template.py (hunk 2 above)
Comment (pablohmontenegro) [reply l10n_ar_tax/models/account_chart_template.py:98]: In my opinion the sequence is for "Retención de Ganancias" and the field invoice_label on taxes is for perceptions on invoices, not for withholdings.

File: l10n_ar_tax/__init__.py (hunk 3 above)
Comment (pablohmontenegro) [reply l10n_ar_tax/__init__.py:26]: We are applying the same logic here https://github.com/ingadhoc/odoo-argentina-ee/blob/18.0/l10n_ar_account_tax_settlement/hooks.py#L10 , monotributista company is identified with chart_template "ar_base". In my opinios it doesn´t matter if cae ar_base is installed or not.

File: l10n_ar_tax/models/account_fiscal_position_l10n_ar_tax.py (hunk 1 above)
Comment (pablohmontenegro) [reply l10n_ar_tax/models/account_fiscal_position_l10n_ar_tax.py:93]: Yes, you are right, I created this pr to delete the line you mentioned https://github.com/ingadhoc/odoo-argentina/pull/1040 .

File: l10n_ar_tax/models/account_fiscal_position_l10n_ar_tax.py (hunk 4 above)
Comment (pablohmontenegro) [reply l10n_ar_tax/models/account_fiscal_position_l10n_ar_tax.py:109]: I put a pdb here https://github.com/ingadhoc/odoo-argentina/blob/18.0/l10n_ar_tax/models/account_chart_template.py#L142-L143 while installing and the code does not enter there. Also duplicated taxes are duplicated when I install l10n_ar_withholding (https://github.com/odoo/odoo/blob/18.0/addons/l10n_ar_withholding/demo/account_demo.py#L35). Here the taxes are duplicated https://github.com/adhoc-cicd/odoo-odoo/blob/18.0/addons/l10n_ar_withholding/demo/account_demo.py#L36:L38
